react-search/my-app/src/Search.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+type Prop = {
+  quotes: string[];
+};
+export function Search({ quotes }: Prop) {
+  const [searchTerm, setSearchTerm] = useState('');
+  const [filtered, setFiltered] = useState(quotes);
+
+  const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value.toLowerCase();
+    setSearchTerm(value);
+    const result = quotes.filter((item) => item.toLowerCase().includes(value));
+    setFiltered(result);
+  };
+
+  return (
+    <div>
+      <input
```

Comment:
The purpose of this exercise is to give you experience getting components to interact with each other. So doing this whole thing as just one component defeats the purpose.

Please break it into multiple components. You should have two children components, one for the input field and one for the list.